unabridged-works-volume-3/original/15-letters.adoc

```diff
@@ -4890,19 +5156,28 @@ yet the reading and serious consideration thereof may not be unserviceable to th
 
 I am thy friend, in true love and desires for thee,
 
+[.signed-section-signature]
 I+++.+++ P.
 
-THE PLAIN PATH-WAY OF SALVATION
+[.letter-heading]
```

Comment:
I wasn't sure about this whole little section of [discourse-part] here either, nor the letterhead above it.

Reply:
Yeah, i'm a bit confused too. I think let's a) keep the _discourse parts_ below, but b) change the heading to be a `===` `.centered, like this:

```ASCIIDOC
[.centered]
=== The Plain Path-way of Salvation
```